### DISCUSSION TITLE
What dose "lp" mean?

### DISCUSSION THREAD

**Discussion by PaulRBerg** [lucas-manuel/invariant-example#5] [Q&A] [answered]
I had a skim through the repo - does "lp" mean "liquidity provider", or something else specific to invariant testing?

Side note - would you be open to open [discussions][1] in this repo, so that I don't flood the issues channel with questions?

[1]: https://docs.github.com/en/discussions

**Comment by lucas-manuel** [comment, marked as answer]
Liquidity Provider yeah 👍 I like the idea of using actor though as in your example you gave in the `book` repo.
